src/client/styles/utils/_base.scss

```diff
@@ -1,83 +1,83 @@
-a {
-  &.button {
-    color: $button-text-color;
-    background: $button-background-color;
-    padding: 0.35rem 0.65rem;
-    text-decoration: none;
-    white-space: nowrap;
-
-    &:hover {
-      color: $button-background-color;
-      background: $button-text-color;
-    }
-  }
-}
-
-button {
-  color: $button-text-color;
-  background: $button-background-color;
-  border: 1px solid $button-background-color;
-
-  &.large {
-    font-size: 1.3rem;
-  }
-}
-.button-group {
-  button {
-    margin-right: 1rem;
-  }
-}
-
-p {
-  &.lead {
-    font-size: 1.4rem;
-    margin: 1rem 0;
-  }
-}
-
-a, input, select, button {
-  border-radius: 0;
-
-  &:focus {
-    outline-color: $focus-color;
-    outline-style: solid;
-    outline-width: $focus-width;
-  }
-}
-
-iframe {
-  width: 100%;
-}
-
-.form {
-  button[type="submit"] {
-    margin-top: 1rem;
-  }
-}
-
-.inline-form {
-  input {
-    border: 1px solid $border-color;
-    padding: 0.2rem 0;
-    text-indent: 0.2rem;
-    width: 12rem;
-  }
-}
-
-.visuallyhidden {
-  border: 0;
-  clip: rect(0 0 0 0);
-  height: 1px;
-  margin: -1px;
-  overflow: hidden;
-  padding: 0;
-  position: absolute;
-  width: 1px;
-}
-
-.container {
-  max-width: $container-width;
-  padding: 0.5rem 1rem;
-  margin: 0 auto;
-  @include box-sizing(border-box);
-}
+// a {
```

Comment:
Remove the code if it's not needed rather than commenting it out.

Reply:
👍